crates/re_viewer/src/misc/caches/mod.rs

```diff
@@ -101,21 +100,48 @@ impl TensorStats {
                 });
             (min.to_f64(), max.to_f64())
         }
+        */
 
         let range = match tensor.dtype() {
-            TensorDataType::U8 => as_ndarray::<u8>(tensor).ok().map(tensor_range_u8),
-            TensorDataType::U16 => as_ndarray::<u16>(tensor).ok().map(tensor_range_u16),
-            TensorDataType::U32 => as_ndarray::<u32>(tensor).ok().map(tensor_range_u32),
-            TensorDataType::U64 => as_ndarray::<u64>(tensor).ok().map(tensor_range_u64),
-
-            TensorDataType::I8 => as_ndarray::<i8>(tensor).ok().map(tensor_range_i8),
-            TensorDataType::I16 => as_ndarray::<i16>(tensor).ok().map(tensor_range_i16),
-            TensorDataType::I32 => as_ndarray::<i32>(tensor).ok().map(tensor_range_i32),
-            TensorDataType::I64 => as_ndarray::<i64>(tensor).ok().map(tensor_range_i64),
-
-            TensorDataType::F16 => as_ndarray::<f16>(tensor).ok().map(tensor_range_f16),
-            TensorDataType::F32 => as_ndarray::<f32>(tensor).ok().map(tensor_range_f32),
-            TensorDataType::F64 => as_ndarray::<f64>(tensor).ok().map(tensor_range_f64),
+            TensorDataType::U8 => ndarray::ArrayViewD::<u8>::try_from(tensor)
```

Comment:
I would add a `use ndarray::ArrayViewD;` to this function to shorten the code below